FusionIIIT/applications/central_mess/migrations/0018_merge_20210303_2016.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.5 on 2021-03-03 20:16
+
+from django.db import migrations
```

Comment:
Don't push migrations. It is already told to not to do this. If you are using local DB, just simply remove these files from the staging area before commit.